apps/explorer/src/pages/transaction-result/TransactionData.tsx

```diff
@@ -40,35 +36,22 @@ export function TransactionData({ transaction }: Props) {
 
     return (
         <div className="flex flex-wrap gap-6">
-            <section className="flex w-96 min-w-[50%] flex-1 flex-col gap-6">
```

Comment:
We will get squished content if we do not have a width for the content

<img width="412" alt="Screenshot 2023-05-06 at 10 03 07 AM" src="https://user-images.githubusercontent.com/127577476/236637451-e43fc254-d3e1-4786-a6f3-c2cd28234d34.png">

Reply:
we should adjust the breakpoint on the parent layout so that it lays out in a column so we don't get to this point. I'll continue to refine this

Reply:
It's because we need either set width here, so that the content will clip behind the drag panel, or we set `minWidth` on the panel, so it will never collapse beyond this (going beyond will become a snap in).